source/includes/_milkyway_api_doc.md

```diff
@@ -0,0 +1,82 @@
+# Milkyway API
+
+## Definition
+
+The goal of this page is to define the high level architecture of the Milkyway API that will allow matching agencies (called here entities) using Milkyway network.
+
+
+## **Populate endpoint**
+
+>Shell command:
+
+```shell
+curl --header "Content-Type: application/json" \
+ --header "Authorization: Bearer <ACCESS_TOKEN>"\
+ --request PUT \
+ --data '{"skills":["<SKILL_1>", "<SKILL_2>", "<SKILL_3>", "<SKILL_4>", "<SKILL_5>"]}' \
+ http://milkyway.sortlist.com/v1/<ENVIRONMENT>/entities/<ENTITY_ID>
+```
+>Example of request:
```

Comment:
there is two times an example of request, keep only one of them

Reply:
i feel like is better to put the http request and the curl command separately